plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -383,11 +378,11 @@ public LibvirtUtilitiesHelper getLibvirtUtilitiesHelper() {
     }
 
     public CPUStat getCPUStat() {
-        return _cpuStat;
+        return new CPUStat();
     }
 
     public MemStat getMemStat() {
-        return _memStat;
+        return new MemStat(_dom0MinMem, _dom0OvercommitMem);
```

Comment:
Same comment as above, the returned object instance is always different for every method call